src/V3Width.cpp

```diff
@@ -4905,6 +4910,34 @@ class WidthVisitor final : public VNVisitor {
                                         // func
         }
     }
+    void visit(AstProperty* nodep) override {
+        if (nodep->didWidth()) return;
+        if (nodep->doingWidth()) {
+            UINFO(5, "Recursive property call: " << nodep);
+            nodep->v3warn(E_UNSUPPORTED,
+                          "Unsupported: Recursive property call: " << nodep->prettyNameQ());
+            nodep->recursive(true);
+            nodep->didWidth(true);
+            return;
+        }
+        nodep->doingWidth(true);
+        m_ftaskp = nodep;
+        // Property call will be replaced by property body in V3AssertPre. Property body has bit
+        // dtype, so set it here too
+        nodep->dtypeSetBit();
+        for (AstNode* propStmtp = nodep->stmtsp(); propStmtp; propStmtp = propStmtp->nextp()) {
+            if (VN_IS(propStmtp, Var))
+                userIterate(propStmtp, nullptr);
+            else if (VN_IS(propStmtp, PropClocked))
+                iterateCheckSizedSelf(nodep, "PropClocked", propStmtp, SELF, BOTH);
+            else
+                propStmtp->v3warn(E_UNSUPPORTED, "Unsupported: " << propStmtp->prettyNameQ()
+                                                                 << " in AstProperty");
```

Comment:
```suggestion
            if (VN_IS(propStmtp, Var)) {
                userIterate(propStmtp, nullptr);
            } else if (VN_IS(propStmtp, PropClocked)) {
                iterateCheckSizedSelf(nodep, "PropClocked", propStmtp, SELF, BOTH);
            } else {
                propStmtp->v3warn(E_UNSUPPORTED, "Unsupported: " << propStmtp->prettyNameQ()
                                                                 << " in AstProperty");
            }
```
Can the `else` actually be triggered by the user? If so, it should be tested, otherwise just do a `v3fatal("Invalid statement")` or something, as it should be unreachable.

Reply:
It should be unreachable. I changed the error.